docs/TROUBLESHOOTING.md

```
@@ -349,16 +349,16 @@ rm reset-test.php
 
 1. **Run diagnostics**:
    ```bash
-   ./bin/check-env > diagnostics.txt
-   php -v >> diagnostics.txt
-   php -m | grep xdebug >> diagnostics.txt
+./bin/check-env > diagnostics.txt
+php -v >> diagnostics.txt
+php -m | grep xdebug >> diagnostics.txt
    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)